contracts/governance/Vesting/VestingRegistryLogic.sol

```diff
@@ -215,6 +225,15 @@ contract VestingRegistryLogic is VestingRegistryStorage {
                 uint256(VestingType.TeamVesting),
                 _vestingCreationType
             );
+
+        vestingDetail[vesting] = Vesting(
```

Comment:
```suggestion
vestingDetails[vesting] = Vesting(
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/79f72c867c17cd1f12ed1e726ee79e2c308a8e9e)